src/main/kotlin/no/nav/syfo/persistence/AktivitetskravDAO.kt

```diff
@@ -137,6 +146,60 @@ class AktivitetskravDAO(
         }
     }
 
+    fun getHistoriskAktivitetsplikt(fnr: String): List<Aktivitetsplikt>? {
+        val query = """
+            SELECT 
+            vurdering.status, 
+            vurdering.arsaker, 
+            vurdering.sist_vurdert,
+            vurdering.siste_vurdering_uuid, 
+            vurdering.created_at,
+            varsel.created_at, 
+            varsel.svarfrist, 
+            varsel.journalpost_id, 
+            varsel.document, 
+            varsel.vurdering_uuid 
+
+            FROM AKTIVITETSKRAV_VURDERING vurdering
+            LEFT JOIN AKTIVITETSKRAV_VARSEL varsel ON vurdering.siste_vurdering_uuid = varsel.vurdering_uuid
+            WHERE vurdering.person_ident = :person_ident
+            AND vurdering.vurdering_uuid = :uuid
+            ORDER BY vurdering.created_at desc, vurdering.sist_vurdert desc NULLS LAST;
+        """.trimIndent()
+
+        val uuid = getRelevantVurderingUuid(fnr)
+
+        val namedParameters = MapSqlParameterSource()
+            .addValue("person_ident", fnr)
+            .addValue("uuid", uuid)
+
+        return try {
+            namedParameterJdbcTemplate.query(query, namedParameters, AktivitetspliktRowMapper())
+        } catch (e: EmptyResultDataAccessException) {
+            null
+        }
+    }
+
+    fun getRelevantVurderingUuid(fnr: String): UUID? {
```

Comment:
Kan vurdere å ha bare 1 spørring i stedet for to, ala

`SELECT vurdering.status,
       vurdering.arsaker,
       vurdering.sist_vurdert,
       vurdering.vurdering_uuid,
       vurdering.siste_vurdering_uuid,
       vurdering.created_at,
       varsel.svarfrist,
       varsel.journalpost_id,
       varsel.document
FROM AKTIVITETSKRAV_VURDERING vurdering
LEFT JOIN AKTIVITETSKRAV_VARSEL varsel ON vurdering.siste_vurdering_uuid = varsel.vurdering_uuid
WHERE vurdering.vurdering_uuid = (SELECT vurdering_uuid
                                  FROM AKTIVITETSKRAV_VURDERING
                                  WHERE person_ident = :person_ident
                                  ORDER BY created_at desc, sist_vurdert desc NULLS LAST
                                  LIMIT 1)
ORDER BY vurdering.created_at desc, vurdering.sist_vurdert desc NULLS LAST;`

Reply:
Godt poeng!